src/Pages/Withdrawals.razor

```diff
@@ -165,23 +166,31 @@
                             }
                         </DisplayTemplate>
                         <EditTemplate>
-                            @if (!_isAmountDisabled)
-                            {
-                                <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args, _isAmountDisabled)">
-                                    <NumericPicker TValue="decimal" Disabled="_isAmountDisabled" Value="@((decimal)context.CellValue)" ValueChanged="(value) => { context.CellValue = value; }" CurrencySymbol="₿ " Max="@(_maxWithdrawal)" Min="_minWithdrawal" Decimals="8">
-                                        <Feedback>
-                                            <ValidationError />
-                                        </Feedback>
-                                    </NumericPicker>
-                                    <FieldHelp>
-                                        @($"Current amount: {Math.Round(PriceConversionService.BtcToUsdConversion((decimal)context.CellValue, _btcPrice), 2)} USD")
-                                    </FieldHelp>
+                                <Validation Validator="args => ValidationHelper.ValidateWithdrawalAmount(args, false)">
+                                    <NumericPicker TValue="decimal" @bind-Value="@_amount" Step="0.00001m" CurrencySymbol="₿ " Max="@(_maxWithdrawal)" Min="_minWithdrawal" Decimals="8" />
+                                     <FieldHelp>
+                                       @if (Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT < _amount)
+                                       {
+                                           <span class="text-danger">@($"Maximum amount {Constants.MAXIMUM_WITHDRAWAL_BTC_AMOUNT} BTC.")</span>
+                                       }
```

Comment:
Same as in channel requests, this should be validated with the validation in line 169.

Reply:
Validation in the case doesn't help because we have to calculate field help everytime when a user input a new value.